models/demos/t3000/mixtral8x7b/tt/mixtral_attention.py

```diff
@@ -154,28 +155,24 @@ def forward_decode(
         xs,
         start_pos,
         current_pos,
-        attn_masks,
         rot_mat,
     ):
         """
         x: (seq_len, 1, batch, hidden_dim)
         start_pos: the length of the KV cache. Same as current token's index.
         current_pos: start_pos
-        attn_masks: (seq_len, batch, n_heads, cache_len+seq_len)
-        rot_mat: rotation matrix for each device
+        rot_mats: list of rotation matrices for each device
```

Comment:
for forward_decode we just pass a single rot_mat. For correctness, can you re-add the correct description? `rot_mat: rotation matrix for each device`.

Can you also add the input description to the `forward_prefill` function for completeness? 🙏